ted2zim/templates/assets/app.js

```diff
@@ -84,7 +84,7 @@ function refreshPagination() {
 	if (pageCount > 1) {
 	    var pageText = document.getElementById('pagination-text');
 	    var pageNumber = videoDB.getPageNumber();
-	    pageText.innerHTML = 'Page ' + pageNumber + '/' + pageCount;
+	    pageText.innerHTML += ' ' + pageNumber + '/' + pageCount;
```

Comment:
what's this change?

Reply:
That's because I now already pass the translated form of the word "Page" in the template. So we just need to update the numbers.